src/spring-cloud/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+3.1.6
+---
+* Command group 'spring-cloud' is deprecated and will be removed in a future release. Use 'spring' instead.
```

Comment:
It is recommended to clearly inform users to install the new extension `spring` in the description

Reply:
updated